src/lib/y2network/interface_config_builders/bridge.rb

```diff
@@ -0,0 +1,40 @@
+require "yast"
+require "y2network/interface_config_builder"
+
+Yast.import "LanItems"
+
+module Y2Network
+  module InterfaceConfigBuilders
+    class Bridge < InterfaceConfigBuilder
+      def initialize
+        super(type: "bridge")
+      end
+
+      def bridgable_interfaces
+        Yast::LanItems.GetBridgeableInterfaces(name)
+      end
+
+      def already_configured?(devices)
+        configurations = Yast::NetworkInterfaces.FilterDevices("netcard")
+        netcard_types = (Yast::NetworkInterfaces.CardRegex["netcard"] || "").split("|")
+
+        confs = netcard_types.reduce([]) do |res, devtype|
+          res.concat((configurations[devtype] || {}).keys)
+        end
+
+        devices.each do |device|
+          next if !confs.include?(device)
+
+          dev_type = Yast::NetworkInterfaces.GetType(device)
+          ifcfg_conf = configurations[dev_type][device]
+
+          next if ifcfg_conf["BOOTPROTO"] == "none"
```

Comment:
```next if [nil, "none"].include? NetworkInterfaces.devmap(device)``` and 5 lines above can be dropped

Reply:
so this one line replace whole that part above?

Reply:
In case we permit to add it to the bond when the BOOTPROTO is not defined, then we should at least adapt the BOOTPROTO when the interface is added to the bond.